documentation/modules/auxiliary/admin/kerberos/inspect_ticket.md

```diff
@@ -0,0 +1,157 @@
+# Kerberos Ticket Inspecting
+
+The `inspect_ticket` module allows you to print the contents of a ccache/kirbi file.
+
+## Pre-Verification steps
+
+1. Generate a ccache file using the `forge_ticket` module
+
+## Verification Steps
+
+1. Start msfconsole
+2. Do: `use auxiliary/admin/kerberos/inspect_ticket`
+3. Do: `set TICKET_PATH /path/to/ccache/file`
+4. Optional: either `set AES_KEY aes_key_here` or `set NTHASH nthash_here`
+5. Do: `run` to see the contents of the ticket 
+
+## Scenarios
+
+### Inspecting Ticket contents
+
+This action allows you to see the contents of any ccache or kirbi file,
+If you are able to provide the decryption key we can also show the encrypted parts of the tickets.
+
+1. `TICKET_PATH` - The path to the ccache or kirbi file.
+2. `AES_KEY` - (Optional) Only set this if you have the decryption key and it is an AES128 or AES256 key.
+3. `NTHASH` - (Optional) Only set this if you have the decryption key and it is an NTHASH.
+No other options are used in this action.
+
+**Without Key**
+
+```
+msf6 auxiliary(admin/kerberos/inspect_ticket) > run TICKET_PATH=/path/to/ticket
+```
+
+Example Output:
+```
```

Comment:
sorry not sure I follow when you say this could be merged into one section?

Reply:
You have two code snippets, separated by the line `Example Output:`, that could likely be merged into one code snippet since the output that follows `Example Output:` is the same output that would appear directly after the first snippet in a normal run of this module.

Therefore the `Example Output:` line seems to be extra filler here that isn't needed and we should take it out and merge these two code snippets into one code snippet.

I'll go ahead and make the changes to do this so long.

Reply:
Fixed in d69564f3dfc1360b24a62c7611924e8a9f4611e6